graf2d/gpadv7/v7/src/RDrawingOptsBase.cxx

```diff
@@ -28,16 +28,19 @@
 std::shared_ptr<ROOT::Experimental::RDrawingAttrHolder> &
 ROOT::Experimental::RDrawingOptsBase::GetHolder()
 {
-   if (!fHolder)
+   if (!fHolder) {
       fHolder = std::make_shared<RDrawingAttrHolder>();
```

Comment:
Don't we need a symmetrical "if I/O has read `fHolderIO`, set `fHolder` from it"?

Reply:
Current  `temprorary` solution works **only** for storage of attributes in JSON. 

There is no way to correctly restore such object from ROOT or JSON file - shared_ptr/weak_ptr problems.